guppylang/std/unsupported.py

```diff
@@ -0,0 +1,354 @@
+# mypy: disable-error-code="empty-body, no-untyped-def"
+"""Python builtins that are not supported yet"""
+
+from pathlib import Path
+
+from guppylang.decorator import ModuleIdentifier, guppy
+from guppylang.std import builtins
+from guppylang.std._internal.checker import UnsupportedChecker
+
+builtins_module = guppy.get_module(
```

Comment:
So can you try making this private: `_builtins_module`?

Reply:
Seems like I still get the same error as I did locally :(